src/pybind/mgr/cephadm/services/iscsi.py

```diff
@@ -58,8 +58,19 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
                 'val': key_data,
             })
 
+        #add mgr ip address to trusted list with dashboard
+        mgr_ips = []
+        trusted_ip_list = spec.trusted_ip_list if spec.trusted_ip_list else ''
+        for dd in self.mgr.cache.get_daemons_by_service('mgr'):
+            assert dd.hostname is not None
+            mgr_ips.append(self.mgr.inventory.get_addr(dd.hostname))
+        if trusted_ip_list:
+            trusted_ip_list += ','
+        trusted_ip_list += ",".join(mgr_ips)
```

Comment:
what happens if the mgr daemon is redeployed on another host? maybe the dashboard should query the daemon state from the orchestrator?

Reply:
iscsi daemons will now be reconfigured when the active mgr changes